ajax/sendMail.json.php

```diff
@@ -1,6 +1,7 @@
 <?php
 require_once(__DIR__ . '/../config.inc.php');
 require_once 'Mail.php';
+require_once "Mail/mime.php";
```

Comment:
Are you sure that you need this line? You don't use any classes/methods from this file.